test/bib-marc-test.js

```diff
@@ -68,6 +68,7 @@ describe('Bib Marc Mapping', function () {
       // that 1) the malformed back-link is treated as such and 2) its presence
       // does not disrupt other parallel field extraction.
       const parallelPlaceOfPublication = bib.parallel('264', ['a', 'b', 'c', 'f', 'g', 'h', 'k', 'n', 'p', 's'])
+      console.log('parallelPlaceOfPublication: ', parallelPlaceOfPublication)
       assert.strictEqual(parallelPlaceOfPublication.length, 0)
```

Comment:
This evaluates to `[null, '']`. It should probably evaluate to `['', '']` for consistency, but the reason it evaluates to `[null, '']` in this edge case is:
 - SierraRecord.parallel loops over the primary values
 - For each primary value, it looks up the parallel value:
   - If there is no $6 in the primary field, [it returns ''](https://github.com/NYPL-discovery/discovery-store-poster/blob/c97730b92d82948e9b3638bedc8433cea8ab1a57/lib/models/sierra-record.js#L97) (This is the case for the second 264 with $c = "©2015".) 
   - If there is a $6 in the primary but it's malformed, [it returns ''](https://github.com/NYPL-discovery/discovery-store-poster/blob/c97730b92d82948e9b3638bedc8433cea8ab1a57/lib/models/sierra-record.js#L102)
   - It then [queries the record for the parallel value](https://github.com/NYPL-discovery/discovery-store-poster/blob/c97730b92d82948e9b3638bedc8433cea8ab1a57/lib/models/sierra-record.js#L105)
   - The [result of that](https://github.com/NYPL-discovery/discovery-store-poster/blob/c97730b92d82948e9b3638bedc8433cea8ab1a57/lib/models/sierra-record.js#L137) is an array of matching varfield values (which should have a single value)
   - If the highly unusual scenario occurs where the valid-looking parallel link in the primary field doesn't actually refer to any parallel field, the function [happens to return `null`](https://github.com/NYPL-discovery/discovery-store-poster/blob/c97730b92d82948e9b3638bedc8433cea8ab1a57/lib/models/sierra-record.js#L140) This is the case with the second 264, whose $6 is "880-03", implying there should be a 264 with $6 = "264-03", but there is not.

Previously, the policy was to collect parallel values and then remove any blank (falsey, technically) values from the end of that list. Now that you've removed the code that does that, falsey values are left untouched. This is all working correctly, but we should adjust the test to expect the last two values to be falsey/blank.

But I think we should also adjust how we communicate this kind of broken 880 link and return `''` [instead of `null`](https://github.com/NYPL-discovery/discovery-store-poster/blob/c97730b92d82948e9b3638bedc8433cea8ab1a57/lib/models/sierra-record.js#L140) for consistency. I don't see a reason to treat them differently.